packages/apps/demo/src/AchievementSearchBar.tsx

```diff
@@ -59,11 +58,12 @@ export const AchievementSearchBar = (props: SearchBarProps) => {
   const { map, selectDecorator, onSelect } = props;
   const [achievements, setAchievements] = useState<AchievementOption[]>([]);
   useEffect(() => {
+    const game = map === 'usa' ? 'ats' : 'ets2';
     Promise.all([
-      fetch('ats-achievements.json').then(
```

Comment:
🤔 maybe should do the same in CitySearchBar, and only load the JSON data for the current map (and rely on browser caching to prevent actual re-downloads on map switches)